src/main/java/joe/commands/InvalidCommand.java

```diff
@@ -0,0 +1,33 @@
+package joe.commands;
+
+import joe.Storage;
+import joe.TaskList;
+import joe.Ui;
+
+/**
+ * Represents a command for handling invalid user inputs.
+ */
+public class InvalidCommand extends Command {
+    private final String msg;
```

Comment:
Perhaps we could use a more intuitive variable name here? Something like `errorMessage`.